packages/core/src/modules/credentials/CredentialsModule.ts

```diff
@@ -625,6 +630,33 @@ export class CredentialsModule<
     return this.getService(credentialExchangeRecord.protocolVersion)
   }
 
+  private registerFeatures(featureRegistry: FeatureRegistry) {
+    featureRegistry.register<Protocol>(
```

Comment:
Maybe a register multiple would be nice? Or register would always take an array? Not sure.

Reply:
Yeah I think it would be useful to support both single and array, as in other parts of the framework.